shared-helpers/src/views/CustomIconMap.tsx

```diff
@@ -14,6 +15,7 @@ export const CustomIconMap = {
   clock: <Clock />,
   home: <HomeModernIcon />,
   envelope: <EnvelopeIcon />,
+  house: <HouseIcon />,
```

Comment:
question: was this icon change in order to keep more in sync with the current Detroit look? I'm curious why the "modern" home isn't used for the home page

Reply:
Good q! It was just a request from Jesse, it's documented [here](https://www.figma.com/design/L8CtCfMy9j6a5Qvfp8KJO0?node-id=1549-15692&m=dev#1186848271)